packages/electrode-react-webapp/lib/react-webapp.js

```diff
@@ -104,14 +104,31 @@ function makeRouteHandler(routeOptions, userContent) {
       }
 
       const chunkNames = chunkSelector(request);
-      const devCSSBundle = chunkNames.css
-        ? `${devBundleBase}${chunkNames.css}.style.css`
-        : `${devBundleBase}style.css`;
-      const devJSBundle = chunkNames.js
-        ? `${devBundleBase}${chunkNames.js}.bundle.dev.js`
-        : `${devBundleBase}bundle.dev.js`;
-      const jsChunk = _.find(assets.js, asset => _.includes(asset.chunkNames, chunkNames.js));
-      const cssChunk = _.find(assets.css, asset => _.includes(asset.chunkNames, chunkNames.css));
```

Comment:
I'm not sure what the purpose of this was except to silently stomp file name changes that would have otherwise worked.